www/index.js

```diff
@@ -152,16 +159,16 @@ HotPush.prototype._hasloadedLocalFile = function() {
 HotPush.prototype.update = function() {
   var self = this;
   if (this.options.type === 'replace') {
-    this._syncs = [ContentSync.sync({ src: this.options.archiveURL, id: 'assets', headers: this.options.headers})];
+    this._syncs = [ContentSync.sync({ src: this.options.archiveURL, id: 'assets', copyCordovaAssets: true, headers: this.options.headers})];
```

Comment:
to make sure the cordova assets folder are copied over to the new folder where the assets.zip will be decompressed. it enables you to completely point the document.location of the app to the new folder instead of just referencing the files in the index.html.
I can also make that an option. You will understand this approach better when I publish my module.